Makefile

```diff
@@ -1155,6 +1157,16 @@ glance_kuttl: namespace input openstack_crds openstack_storage_crds deploy_clean
 	make deploy_cleanup
 	make cleanup
 
+.PHONY: horizon_kuttl_run
+horizon_kuttl_run: ## runs kuttl tests for the horizon operator, assumes that everything needed for running the test was deployed beforehand.
+	kubectl-kuttl test --config ${HORIZON_KUTTL_CONF} ${HORIZON_KUTTL_DIR}
+
+.PHONY: horizon_kuttl
+horizon_kuttl: namespace input openstack_crds openstack_storage_crds deploy_cleanup mariadb mariadb_deploy keystone keystone_deploy horizon_deploy_prep horizon ## runs kuttl tests for the horizon operator. Installs openstack and openstack-storage crds, mariadb, keystone and horizon operators and cleans up previous deployments before running the tests and, add cleanup after running the tests.
```

Comment:
horizon requires infra operator for memcached. Probably we should add it to the dependency ?

Reply:
Does it also need `memcached_deploy` then?

Reply:
Also, `cleanup` and `deploy_cleanup` are missing includes for Horizon, Infra and Memcached.  I think they would need them for these new targets to clean properly.